src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -1862,7 +1862,7 @@ private IEnumerable<PENamedTypeSymbol> CreateNestedTypes()
 
             foreach (var typeRid in nestedTypeDefs)
             {
-                if (module.ShouldImportNestedType(typeRid))
+                if (module.ShouldImportNestedType())
```

Comment:
Looks like even `module` isn't used. Can we remove this method as whole?

If not and it's kept without passing `typeRid`, then consider calling it once outside the `foreach`

Reply:
I'd kept the method because of the comment it holds, but from discussion with Chuck I'll move the comment here.